### PR TITLE
Feature/devsu 2144 column names update

### PR DESCRIPTION
**Pull request by elewis2**
Updates column name for cancerRelated -> kbStatementRelated
Adds new column to IprGene type (cancerGeneListMatch)

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -109,6 +109,6 @@ def test_found_tumour_supressor(self, report_upload_content: Dict) -> None:
         genes = report_upload_content['genes']
         assert any([g.get('tumourSuppressor', False) for g in genes])
 
-    def test_found_cancer_related_gene(self, report_upload_content: Dict) -> None:
+    def test_found_kb_statement_related_gene(self, report_upload_content: Dict) -> None:
         genes = report_upload_content['genes']
-        assert any([g.get('cancerRelated', False) for g in genes])
+        assert any([g.get('kb_statement_related', False) for g in genes])
```
Comment (mathieulemieux) [inline tests/test_main.py:114]: Why kb_statement_related instead of kbStatementRelated ?

File: setup.cfg [hunk 2]
```diff
@@ -33,7 +33,7 @@ dependency_links = []
 include_package_data = True
 install_requires =
     graphkb>=1.11.0
```
Comment (dustinbleile) [inline setup.cfg:35]: Would this need the pori_graphkb_python currently in PR?

graphkb>1.14.0

[the remark this answers is not in the document]
Comment (elewis2) [reply setup.cfg:35]: it should be backwards compatible.

File: tests/test_main.py (hunk 1 above)
Comment (elewis2) [reply tests/test_main.py:114]: my mistake!

**Pull request merged**